templates/api-template-go/configs/dev-config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "Host":"Test",
```

Comment:
please add config values for prod and staging

Reply:
Updated